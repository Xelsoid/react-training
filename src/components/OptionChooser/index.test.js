import React from 'react';
import ComponentExample from './index';

describe('OptionChooser test', () => {
  const optionsConfig = [
    {
      radioBtnName: 'RELEASE DATE', name: 'sortBy', value: 'releaseDate',
    },
    {
      radioBtnName: 'RATING', name: 'sortBy', value: 'rating',
    },
  ];
  it('renders correctly', () => {
    const component = shallow(
      <ComponentExample
        optionsConfig={optionsConfig}
        defaultValue={optionsConfig[0].value}
        onChangeCallback={() => {}}
      />,
    );
    expect(component).toMatchSnapshot();
  });
});
