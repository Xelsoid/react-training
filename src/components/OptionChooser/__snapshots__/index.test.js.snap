// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionChooser test renders correctly 1`] = `
<div
  className="option-chooser-wrapper"
>
  <input
    checked={true}
    className="option-chooser__option"
    id="releaseDate"
    name="sortBy"
    onChange={[Function]}
    type="radio"
    value="releaseDate"
  />
  <label
    className="option-chooser__label"
    htmlFor="releaseDate"
  >
    RELEASE DATE
  </label>
  <input
    checked={false}
    className="option-chooser__option"
    id="rating"
    name="sortBy"
    onChange={[Function]}
    type="radio"
    value="rating"
  />
  <label
    className="option-chooser__label"
    htmlFor="rating"
  >
    RATING
  </label>
</div>
`;
